modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -55,6 +62,18 @@ const cloneFormats = function (oldCell: SugarElement, newCell: SugarElement, for
   }).getOr(newCell);
 };
 
+const copyAppropriateAttributes = <T extends HTMLElement> (original: SugarElement<T>, clone: SugarElement<T>): void => {
```

Comment:
This still hasn't been fixed as per my previous unresolved comment. If you're dead set on copy, please provide some justification.

Reply:
What? It was transfer before, then I changed it to copy ( Instead of clone, by mistake ). I can't find any comment about you pointing out that it was copy before. But yes, I'll change it.

Reply:
I've been informed this was because github was witholding that particular comment from me. My bad.